client/js/components/document/DpMapSettingsPreview.vue

```diff
@@ -387,7 +387,7 @@ export default {
 
     isNotEmptyFeatureCollection (string) {
       try {
-        return JSON.parse(string).features.length > 0
+        return Object.keys(JSON.parse(string).features).length > 0
```

Comment:
is features always an object? I have in mind that sometimes it is also an array and we have not yet been able to find out where this comes from.
```suggestion
        const parsedFeatures = JSON.parse(string).features
        return Array.isArray(parsedFeatures) ? parsedFeatures.length > 0 : Object.keys(JSON.parse(string).features).length > 0
```